src/webview/android/binding.rs

```diff
@@ -0,0 +1,153 @@
+use crate::http::{
+  header::{HeaderMap, HeaderName, HeaderValue},
+  status::StatusCode,
+};
+use tao::{
+  platform::android::ndk_glue::jni::{
+    errors::Error as JniError,
+    objects::{JClass, JMap, JObject, JString},
+    sys::jobject,
+    JNIEnv,
+  },
+  window::Window,
+};
+
+use super::{MainPipe, WebViewMessage, IPC, REQUEST_HANDLER};
+
+pub struct WebResourceRequest {
+  /// The request url.
+  pub url: String,
+  /// The request method.
+  pub method: String,
+  /// The request headers.
+  pub headers: HeaderMap<HeaderValue>,
+}
+
+pub struct WebResourceResponse {
+  /// The response's status
+  pub status: StatusCode,
+
+  /// The response's headers
+  pub headers: HeaderMap<HeaderValue>,
+
+  /// The response's mimetype type
+  pub mimetype: Option<String>,
+
+  /// The response body.
+  pub body: Vec<u8>,
+}
```

Comment:
Since we move logics to wry, I think we can just use `http` crate's request and response?

Reply:
Totally! I'll change that later.